addons/mail/static/src/core/common/composer.js

```diff
@@ -539,9 +539,20 @@ export class Composer extends Component {
             // Reset signature when recovering an empty body.
             composer.emailAddSignature = true;
         }
+        let signature = this.store.self.signature;
```

Comment:
And if necessary, `signature` coming from here should be flagged as markup beforehand if not already the case (when inserting the field from the server).

Reply:
Not untrue. I keep the r+ to try to close the freeze and avoid "yet another turn" (well, not sure the mergebot will handle much more today anyway), will try to remember to change that after freeze rush once everything is in.